packages/client/src/components/ClientApp.svelte

```diff
@@ -128,6 +128,7 @@
     class:builder={$builderStore.inBuilder}
     class:show={fontsLoaded && dataLoaded}
   >
+    <div class="modal-portal" />
```

Comment:
We already have a `modal-container` node further down (line 210) that we use for all normal modals. Can we not just use that here too?
We would just need to update the class name inside your new modal component as it also uses the same class name.